### DISCUSSION TITLE
kitty 终端无法输入中文问题

### DISCUSSION THREAD

**Discussion by pzzxqj** [fcitx/fcitx5#734] [Q&A]
我搜索到 https://github.com/fcitx/fcitx5/discussions/628#discussion-4503444 ，尝试设行`env GLFW_IM_MODULE=ibus kitty`，但出现了以下报错，且 kitty 不能使用fcitx5。

```shell
xkbcommon: ERROR: couldn't find a Compose file for locale "zh_CN.utf-8" (mapped to "zh_CN.utf-8")
[060 13:49:47.312341] [glfw error 65544]: Failed to create XKB compose table for locale zh_CN.utf-8
```

设置`LANG=en_US.utf-8`也是类似错误，设置`LANG=C`，不再报错，但仍然不能使用输入法。

另外我使用 kde wayland

**Comment by samwhelp** [comment]
若是 Locale 相關的議題

> 我猜測您需要先「`locale-gen`」

## Debian or Ubuntu

``` sh
sudo locale-gen zh_CN.UTF-8
```

* Ubuntu Wiki / [Locale](https://help.ubuntu.com/community/Locale)


## Arch

先編輯「`/etc/locale.gen`」這個檔案

然後再執行下面指令

``` sh
sudo locale-gen
```

* Arch Wiki / Locale / [Generating locales](https://wiki.archlinux.org/title/locale#Generating_locales)

## 注意事項

> 另外可能要注意一下「大小寫」，我不確定是否有影響

``` sh
LANG="zh_CN.UTF-8"
``` 

## 我的設定腳本 (fcitx5-chewing)

* [Ubuntu](https://github.com/samwhelp/note-about-ubuntu/blob/gh-pages/_demo/adjustment/env/im/fcitx5-chewing/config-install.sh)
* [Arch](https://github.com/samwhelp/ezarcher-adjustment/blob/main/prototype/env/im/fcitx5-chewing/config-install.sh)

## Arch Wiki

* Fcitx5 / [Integration](https://wiki.archlinux.org/title/Fcitx5#Integration)
